docs/spark-ddl.md

```diff
@@ -306,12 +306,17 @@ ALTER TABLE prod.db.sample ALTER COLUMN col FIRST
 ALTER TABLE prod.db.sample ALTER COLUMN nested.col AFTER other_col
 ```
 
-Nullability can be changed using `SET NOT NULL` and `DROP NOT NULL`:
+Nullability for a non-nullable column can be changed using `DROP NOT NULL`:
 
 ```sql
 ALTER TABLE prod.db.sample ALTER COLUMN id DROP NOT NULL
 ```
 
+{{< hint info >}}
+It is not possible to change a nullable column to non-nullable with `SET NOT NULL` because Iceberg doesn't know whether there is existing data with null values.
```

Comment:
[optional] 
```suggestion
It is not possible to change a nullable column to a non-nullable column with `SET NOT NULL` because Iceberg doesn't know whether there is an existing data with null values.
```

Reply:
addressed the first part, but `...there is existing data with...` is gramatically correct and having `..an existing data...` would rather make it incorrect